package.json

```diff
@@ -15,6 +15,7 @@
     "url": "https://github.com/fastify/fast-uri/issues"
   },
   "homepage": "https://github.com/fastify/fast-uri",
+  "sideEffects": false,
```

Comment:
What is this? How is this used by the project?

Reply:
1. https://wiki.commonjs.org/wiki/Packages/1.0
2. https://docs.npmjs.com/cli/v10/configuring-npm/package-json?v=true

In general, if it isn't documented in one of those places, and doesn't have a direct use by the project itself (e.g. a `precommit` field used by `@fastify/pre-commit`), I don't think unexplained properties should be added to the package manifest. If it requires searching the Internet to figure out its purpose, it is an unmaintainable property contributing nothing other than mystery to the project.

Reply:
I agree with this in principle

Reply:
None of which are used by this project.